bootstrap/core/lodestar-apps/lodestar-hostiing.yaml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
filename has an extra `i` - won't match the `kustomization.yaml` entry

Reply:
renamed